README.md

```diff
@@ -154,6 +154,28 @@ kubectl delete namespace kogito-workflows
 make undeploy
 ```
 
+## Use Workflow Development Mode
+In the development mode, a user can edit and reload the files using the quarkus dev mode.
```

Comment:
need to sync this with the work that @davidesalerno is doing about exposing the dev mode switch in the CR.

Reply:
There aren't conflicts